core-services/prow/02_config/_plugins.yaml

```diff
@@ -1860,6 +1864,18 @@ config_updater:
         - ci
       gzip: true
       name: ci-operator-misc-configs
+    ci-operator/config/**/log-file-metric-exporter/ViaQ-log-file-metric-exporter-feature*.yaml:
```

Comment:
you can remove this "feature" section as this was an example of one specific branch we never used afterwards